Source/Web/package.json

```diff
@@ -5,15 +5,17 @@
   "main": "karma.conf.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "build": "webpack --mode production",
-    "start": "webpack-dev-server --mode development --open --watch --progress --hot",
+    "build": "webpack --mode production --env.DOLITTLE_ENVIRONMENT=production",
```

Comment:
Nice..  Added an issue to get this into the boilerplate as well: https://github.com/dolittle-boilerplates/Interaction.Web.JS.Aurelia/issues/7